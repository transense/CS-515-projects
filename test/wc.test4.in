12
123

456

78
910 eleven twelve !!