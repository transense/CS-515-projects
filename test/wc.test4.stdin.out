     7   8   37